docs/en_US/TrainingService/LocalMode.rst

```diff
@@ -17,8 +17,9 @@ You have an implementation for MNIST classifer using convolutional layers, the P
 
 To enable NNI API, make the following changes:
 
-* Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
-* Get predefined parameters
+   1.1 Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
```

Comment:
I found the first sentence of this file has typo: "In this tutorial, we will use the example in [~/examples/trials/mnist-tfv1] to explain how to create and run an experiment on local with NNI API."

Reply:
[nni/examples/trials/mnist-tfv1] and NNI APIs?

Reply:
In this tutorial, we will use the example in [~/examples/trials/mnist-tfv1] to explain how to create and run an experiment on local with NNI API.